lib/events-scraper/event.rb

```diff
@@ -52,7 +52,7 @@ def store_file
     # check if the events directory is exists.
     Dir.mkdir(EVENTS_DIR) unless File.exists?(EVENTS_DIR)
     # store the events files.
-    File.open("#{EVENTS_DIR}#{@date_string}.html", "w:UTF-8") do |f|
+    File.open("#{EVENTS_DIR}#{@date_string}.html.erb", "w:UTF-8") do |f|
```

Comment:
I will fix this in another branch, this branch is called "intendation-and-erb" .